templates/ammos-cubs-ait.template.yaml

```diff
@@ -125,12 +219,12 @@ Resources:
           function exitTrap(){
           exitCode=$?
           /opt/aws/bin/cfn-signal \
-          --stack ${AWS::StackName} \
+          --stack ast-sandbox-main-AmmosCubsAITStack-1BQ7JZPVM3HIB \
```

Comment:
🙅 no hard-coding
in fact, this won't even work for subsequent deployments since the stack ID gets generated for new deploys